src/app/learn/get-started/advanced/sticky-settings/integrations/skyux-components/flyouts/index.html

```diff
@@ -0,0 +1,29 @@
+<stache
+  pageTitle="Sticky Settings for Flyouts"
+  navTitle="Flyouts">
+
+  <stache-page-summary>
+    <p>
+      To save user configuration settings in flyouts, add the <sky-code>settingsKey</sky-code> property to the <sky-code>config</sky-code> parameter of the <sky-code>open(component, config?)</sky-code> property that the flyout service uses to open the flyout. The <sky-code>settingsKey</sky-code> must be unique to the component.
+    </p>
+    <p>
+      The UI Config Service preserves the width of flyouts.
+    </p>
+    <stache-include fileName="sticky-settings-whitelist.html"></stache-include>
+
+    <sky-code-block languageType="typescript">
+       public openFlyoutWithCutsomWidth(): void {
+        const flyoutConfig: SkyFlyoutConfig = {
+          defaultWidth: 350,
+          maxWidth: 500,
+          minWidth: 200,
+          settingsKey: my-flyout
```

Comment:
`my-flyout` needs quotes.

Reply:
Good catch. Added this locally and will push the change shortly.